### PR TITLE
chore: Upgrade guide for 0.13

### PR DESCRIPTION
**Pull request by mutahhir**

### CODE REVIEW

File: website/docs/cdktf/release/upgrade-guide-v0-13.mdx [hunk 1]
```diff
@@ -0,0 +1,261 @@
...
+
+```typescript
+import { Image } from "@cdktf/provider-docker/lib/image";
+import { DockerProvider } from "@cdktf/provider-docker/lib/provider";
+import { Container } from "@cdktf/provider-docker/lib/container";
+```
+
+#### Go
+
+<!-- Talk to people about our move to cdktf -->
```
Comment (mutahhir) [inline website/docs/cdktf/release/upgrade-guide-v0-13.mdx:47]: TODO @mutahhir

File: website/docs/cdktf/release/upgrade-guide-v0-13.mdx [hunk 2]
```diff
@@ -0,0 +1,261 @@
+---
+page_title: Upgrading to CDKTF Version 0.13
+description: >-
+  Introducing namespaces for generated providers
+---
+
+# Upgrading to CDK for Terraform Version 0.13
+
+0.13 includes performance improvements to generated providers. Instead of exporting a flat list of exports
+including all supported resources and data sources, we now logically group contructs in namepsaces.
```
Comment (ansgarm) [inline website/docs/cdktf/release/upgrade-guide-v0-13.mdx:10]: > we now logically group contructs in namepsaces

I'm wondering whether "logically" sounds better than it actually is 😁
btw: I think we should call out somewhere that each resource has its own namespace? Or is that in the blog post / other docs?

**Review by ansgarm — COMMENTED** [review]
Couple o' nits only 👍

File: website/docs/cdktf/release/upgrade-guide-v0-13.mdx (hunk 2 above)
Comment (mutahhir) [reply website/docs/cdktf/release/upgrade-guide-v0-13.mdx:10]: Hmm, that's a good point. I'll revise this a bit.

File: website/docs/cdktf/release/upgrade-guide-v0-13.mdx [hunk 3]
```diff
@@ -0,0 +1,261 @@
...
+Due to this, the way you import constructs from your CDKTF application will change.
+
+### CDKTF get and compatibility with non-namespaced providers
+
+CDKTF 0.13 is backwards compatible and supports 0.12 generated (and pre-built) non-namespaced providers without
+any modifications. However, it will only generate namespaced providers. This means that if you were to generate
+a new provider using version 0.13, it will be namespaced. This allows you to gradually migrate your codebase to
+the new namespaced providers gradually.
```
[the remark this answers is not in the document]
Comment (mutahhir) [reply website/docs/cdktf/release/upgrade-guide-v0-13.mdx:18]: Good point.

File: website/docs/cdktf/release/upgrade-guide-v0-13.mdx [hunk 4]
```diff
@@ -0,0 +1,260 @@
...
+page_title: Upgrading to CDKTF Version 0.13
+description: >-
+  Introducing namespaces for generated providers
+---
+
+# Upgrading to CDK for Terraform Version 0.13
+
+0.13 includes performance improvements to generated providers. Instead of exporting a flat list of exports
+including all supported resources and data sources, we now export each construct and it's associated structures in their own namespace.
+Due to this, the way you import constructs from your CDKTF application will change.
```
Comment (ansgarm) [inline website/docs/cdktf/release/upgrade-guide-v0-13.mdx:11]: We should add the redirected link to the blogpost / interim issue containing the reasoning for this change in this paragraph.

**Review by ansgarm — APPROVED** [review]
🚀

**Pull request merged**